hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/operations/IsStillExecutingOperation.java

```diff
@@ -32,13 +32,14 @@
 public class IsStillExecutingOperation extends AbstractOperation implements UrgentSystemOperation {
 
     private long operationCallId;
+    private long operationPartitionId;
 
     IsStillExecutingOperation() {
     }
 
     public IsStillExecutingOperation(long operationCallId, int partitionId) {
         this.operationCallId = operationCallId;
-        setPartitionId(partitionId);
+        this.operationPartitionId = partitionId;
```

Comment:
You still need to point to the right partition in the run method. And can you rename 'partitionId' to 'operationPartitionId'? Just to prevent any confusion.